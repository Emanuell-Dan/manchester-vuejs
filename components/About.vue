<template>
  <div 
    id="about"
    class="pt-16">
    <h2 class="about__title text-vue-brand text-center mobile:text-xl text-4xl mb-8">What we're about</h2>
			
    <div class="about__content text-base">
      <p class="mb-4">Welcome,</p>
      <p class="mb-4">Organised and sponsored by the MoneySuperMarket team in Manchester, this group will run monthly events.
      This will offer a great opportunity for both developers with or without experience in this stack.
      The format of the meetup may differ each month, to give both developers mentioned previously a chance to learn new things and challenge themselves.</p>
      <p class="mb-4">Suggestions for improvements or new topics are always welcomed!</p>
      <p class="mb-4">Should you have something interesting to talk about, either from a theoretical point of view or a demo to showcase,
      this will be your chance! Get in touch with the organiser for more details.</p>

      <div class="tablet:flex tablet:justify-between">
        <p class="mb-4">Thanks and I hope you will all be joining us,<br>Emanuell &amp; the MoneySuperMarket team</p>
        <div class="mobile:pt-8 about__social flex items-center">
          <p class="font-bold mr-4">Find us on:</p>
          <ul class="list-reset flex items-center">
            <li class="mr-4">
              <a
                href="https://www.meetup.com/VueJS-Manchester/"
                target="_blank">
                <img
                  class="cursor-pointer"
                  src="@/static/meetup.png" 
                  alt="Meetup logo">
              </a>
            </li>
            <li>
              <a
                href="https://twitter.com/vuejsmcr"
                target="_blank">
                <img 
                  class="cursor-pointer"
                  src="@/static/twitter.png" 
                  alt="Twitter logo">
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>
</template>

<script>
export default {
	name: 'About'
};
</script>
