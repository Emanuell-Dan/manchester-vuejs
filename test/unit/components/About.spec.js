import About from '@/components/About.vue';
import { mount } from '@vue/test-utils';

describe('About.vue', () => {
	let cmp;

	beforeEach(() => {
		cmp = mount(About);
	});

	it('should render with expected html structure', () => {
		expect(cmp.element).toMatchSnapshot();
	});
});
