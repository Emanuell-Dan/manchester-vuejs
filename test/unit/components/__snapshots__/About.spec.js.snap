// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About.vue should render with expected html structure 1`] = `
<div
  class="about pt-16"
  id="about"
>
  <h2
    class="about__heading font-bold text-center text-2xl md:text-4xl mb-4 md:mb-8"
  >
    
    What we're about
  
  </h2>
   
  <div
    class="about__content text-base"
  >
    <p
      class="about__text mb-4"
    >
      
      Welcome,
    
    </p>
     
    <p
      class="about__text mb-4"
    >
      
      Organised and sponsored by the MoneySuperMarket team in Manchester, this group will run monthly events.
      This will offer a great opportunity for both developers with or without experience in this stack.
      The format of the meetup may differ each month, to give both developers mentioned previously a chance to learn new things and challenge themselves.
    
    </p>
     
    <p
      class="about__text mb-4"
    >
      
      Suggestions for improvements or new topics are always welcomed!
    
    </p>
     
    <p
      class="about__text mb-4"
    >
      
      Should you have something interesting to talk about, either from a theoretical point of view or a demo to showcase,
      this will be your chance! Get in touch with the organiser for more details.
    
    </p>
     
    <div
      class="about__footer md:flex md:justify-between"
    >
      <p
        class="about__greet mb-4"
      >
        
        Thanks and I hope you will all be joining us,
        <br />
        Emanuell & the MoneySuperMarket team
      
      </p>
       
      <div
        class="about__social pt-2 md:pt-0 flex items-center"
      >
        <p
          class="about__contact font-bold mr-4"
        >
          
          Find us on:
        
        </p>
         
        <ul
          class="about__social-list list-reset flex items-center"
        >
          <li
            class="about__social-item mr-4"
          >
            <a
              class="about__social-item--meetup cursor-pointer"
              href="https://www.meetup.com/VueJS-Manchester/"
              target="_blank"
            />
          </li>
           
          <li
            class="about__social-item about__social-item--twitter"
          >
            <a
              class="about__social-item--twitter cursor-pointer"
              href="https://twitter.com/vuejsmcr"
              target="_blank"
            />
          </li>
        </ul>
      </div>
    </div>
  </div>
</div>
`;
